### PR TITLE
Enable Bridge certificate configuration

### PR DESCRIPTION
**Pull request by roncain**
This PR allows the Bridge to install and uninstall certificates
based on the resources that need them.

Specific changes are:
  - Hard-coded knowledge of specific thumbprints removed
  - New .cer and .pfx files generated for localhost
  - New TestProperties and BridgeConfiguration properties
    to make these certificate names configurable
  - New CertificateManager class to manage installing and uninstalling

### CODE REVIEW

**Comment by roncain** [comment]
Reviewers: getting this to work across machines will happen in multiple passes.  This PR is scoped to getting the current self-hosted scenario to pass all tests.  After this is merged, I will issue another PR to generate certs on-demand for cross-machine scenarios (and try to remove netsh process invokes).

All feedback welcome, but please help me by indicating whether your feedback needs to be addressed for this PR to be accepted vs. a suggestion for how to improve something in the next PR.

**Comment by dmetzgar** [comment]
LGTM

File: src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WcfTestBridgeCommon
+{
+    public class CertificateManager
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs]: NIT: Could be made a static class

**Comment by iamjasonp** [comment]
Ah, dang, my comment didn't go through the first time. 

For whatever it's worth now, LGTM :smile:

File: src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                store.Add(cert);
+                store.Close();
+
+                s_myCertificates[cert.Thumbprint] = certificateName;
+                return cert.Thumbprint;
+            }
+        }
+
+        private static void UninstallAllRootCertificates()
+        {
```
Comment (mconnew) [inline src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs]: Although not a blocker for this PR, I would like to see it in a follow up one. We should have some mechanism to identify left behind certificates from a previous run to be able to clean them up. Similar to how the firewall rules are done.

**Pull request merged**